dove/tests/cmd_dove_ct_args.rs

```diff
@@ -0,0 +1,179 @@
+        if folder.to_str().unwrap().find("dove").is_none() {
+            folder.push("dove");
+        }
+        folder
+    };
+    // Project name and path
+    let project_name = "demoproject_22";
+    let project_folder = {
+        let mut folder = dove_folder.clone();
+        folder.push(project_name);
+        if folder.exists() {
+            remove_dir_all(&folder).expect(&format!(
+                "[ERROR] Couldn't delete project directory: {}",
+                folder.to_str().unwrap()
+            ));
+        }
+        folder
+    };
+
+    // $ cargo run -- new demoproject_22
+    // $ dove new demoproject_22
+    {
+        let args = &["dove", "new", project_name];
+        let command_string: String = args.join(" ").to_string();
+        execute(args, dove_folder.clone()).expect(&format!("[COMMAND] {}", &command_string));
+        set_dependencies_local_move_stdlib(&project_folder);
+    }
+
+    // $ cargo run -- build
+    // $ dove build
+    {
+        let args = &["dove", "build"];
+        let command_string: String = args.join(" ").to_string();
+        execute(args, project_folder.clone()).expect(&format!(
+            "[COMMAND] {}\r\n[FOLDER] {}",
+            &command_string,
+            project_folder.to_str().unwrap()
+        ));
+    }
+
+    // project_folder/scripts/sdemo.move
+    {
+        let mut path = project_folder.clone();
+        path.push("scripts");
+        path.push("sdemo.move");
+        write_all(
+            &path,
+            "script {
+                        fun main(_a1:u64,_a2:u64) { }
+                    }",
+        )
+        .unwrap();
+    }
+    // $ cargo run -- ct -a 1 2
+    // $ dove ct -o 1 2
+    {
+        let args = &["dove", "ct", "-a", "1", "2"];
+        let command_string: String = args.join(" ").to_string();
+        execute(args, project_folder.clone()).expect(&format!(
+            "[COMMAND] {}\r\n[FOLDER] {}",
+            &command_string,
+            project_folder.to_str().unwrap()
+        ));
+
+        let tx_path = {
+            let mut file = project_folder.clone();
+            file.push("target");
+            file.push("transactions");
+            file.push("main.mvt");
+            file
+        };
+
+        assert!(
+            tx_path.exists(),
+            "Transaction not found: {}\r\n[Command] {}",
+            tx_path.to_str().unwrap(),
+            &command_string,
+        );
+        let tx_fmt = format!(
+            "{:?}",
+            bcs::from_bytes::<Transaction>(std::fs::read(&tx_path).unwrap().as_ref()).unwrap()
+        );
+
+        assert!(tx_fmt.contains(" args: [U64(1), U64(2)]"));
+        assert!(tx_fmt.contains(" type_args: []"));
+        assert!(tx_fmt.contains(" signers_count: 0"));
+
+        remove_file(&tx_path).unwrap();
+    }
+    // $ cargo run -- ct 'main(1,2)'
+    // $ dove ct 'main(1,2)'
+    {
+        let args = &["dove", "ct", "main(1,2)"];
```

Comment:
Is there any difference in bytecode between `-a 1 2` and `main(1,2)`? This tests suggests there is no.